### DISCUSSION TITLE
Skipping versions when upgrading CZERTAINLY

### DISCUSSION THREAD

**Discussion by ghost** [OmniTrustILM/ilm#43] [Q&A] [answered]
Hello,
is it safe to upgrade CZERTAINLY straight from 2.11.0 to 2.13.0? Or in general, is it safe to skip versions when upgrading? For example from 2.10.0 to 2.13.0 and so on...?

**Comment by 3keyroman** [comment, marked as answer]
Hello @doloban , it should be safe to upgrade from 2.11.0 to 2.13.0.

You should always check [Helm Upgrading](https://docs.czertainly.com/docs/certificate-key/installation-guide/deployment/deployment-helm/upgrading) in case you are using Helm chart, or [VA Upgrading](https://docs.czertainly.com/docs/certificate-key/installation-guide/deployment/deployment-appliance/operations#upgrading) in case you have appliance.

**Comment by ghost** [comment]
Thanks.

Also, to not start a new discussion, I will ask here - does the upgrade to 2.13.0 automatically rewrite the pyadcs connector version in Helm Chart in pyadcs-connector-deployment pod? There is currently version 1.1.1, does the upgrade automatically rewrite the version to 1.1.2? It should, but just wanted to be sure.

**Comment by 3keyroman** [comment]
Yes, it Helm chart version 2.13.0 has preconfigured version of `pyadcs-connector` 1.1.2. This version will be used by default if you have not overwritten it in your `values.yaml`. See the [`Chart.yaml`](https://github.com/CZERTAINLY/CZERTAINLY-Helm-Charts/blob/2.13.0/charts/czertainly/Chart.yaml) of version 2.13.0.
